admin-console/src/main/webapp/src/components/EventTypes/EventTypeTable.tsx

```diff
@@ -101,7 +103,8 @@ const EventTypeTableImpl: React.FunctionComponent<EventTypeTableImplProps> = pro
                     />
                     <Td>{ eventType.displayName }</Td>
                     <Td>{ eventType.name }</Td>
-                    <Td><InstantEmailCell eventType={ eventType } /></Td>
+                    <Td><InstantEmailCell eventType={ eventType }
+                        onClick={ () => props.onCreateInstantTemplate } /></Td>
```

Comment:
This is always passing `onCreateInstantTemplate` to the onclick - I guess it should pass both somehow? Or have a single function that accepts an optional instantTemplate.